backend.py

```diff
@@ -272,7 +272,10 @@ def get_robots_to_start(board):
     starting_coordinates = get_starting_coordinates(board)
     robot_paths = get_robot_paths()
     robots_start = []
-    robot_info = set_new_robot()
+    lifecount = set_new_robot()
+    flagcount = set_new_robot()
+    injurycount = set_new_robot()
+    
```

Comment:
Funkce `set_new_robot` vždycky vrací trojici čísel `(3, 0, 0)`.

Tenhle kód nastaví každou ze tří proměnných na to, co vrátí funkce `set_new_robot()`. Dělá tedy něco jako:
```python
    lifecount = (3, 0, 0)
    flagcount = (3, 0, 0)
    injurycount = (3, 0, 0)
```